src/ui.button-toggle-item/useAttrs.js

```diff
@@ -1,40 +1,22 @@
 import { computed, toValue } from "vue";
 import useUI from "../composables/useUI.js";
-import { cva } from "../utils/utilUI.js";
+
 import defaultConfig from "./config.js";
 
 export default function useAttrs(props, { isSelected, separated, variant }) {
-  const { config, getAttrs, isSystemKey, hasSlotContent, isCVA } = useUI(
-    defaultConfig,
-    () => props.config,
-  );
-
-  const attrs = {};
-
-  for (const key in defaultConfig) {
-    if (isSystemKey(key)) continue;
-
-    const classes = computed(() => {
-      let value = config.value[key];
-
-      if (isCVA(value)) {
-        value = cva(value)({
-          ...props,
-          variant: toValue(variant),
-          separated: toValue(separated),
-          selected: isSelected.value,
-        });
-      }
+  const { config, getKeysAttrs, hasSlotContent } = useUI(defaultConfig, () => props.config);
 
-      return value;
-    });
+  const mutatedProps = computed(() => ({
+    variant: toValue(variant),
+    separated: toValue(separated),
+    selected: isSelected.value,
```

Comment:
selected is not a prop. So you need to add base and extend below and remove related `compoundVariants` logic from the config.